### PR TITLE
Update demo dockerfiles path

### PR DESCRIPTION
**Pull request by rayanimesh**
Closes issue #46.

### CODE REVIEW

**Comment by rayanimesh** [comment]
@bmbouter : Can you please have a look at this PR.

File: _posts/2015-05-21-use-docker-to-try-pulp.md [hunk 1]
```diff
@@ -12,12 +12,12 @@ categories:
 Using Docker, you can now easily and quickly stand up a demo deployment of Pulp. Excluding the time it takes Docker to download the images, it takes only a few seconds to have a brand new deployment running.
 
     
-    $ wget https://raw.githubusercontent.com/pulp/pulp_packaging/master/dockerfiles/centos/start.sh
+    $ wget https://raw.githubusercontent.com/pulp/packaging/docker/dockerfiles/centos/start.sh
```
Comment (bmbouter) [inline _posts/2015-05-21-use-docker-to-try-pulp.md]: This line has a carriage return + line feed (CRLF) at the end of it when all of our code only has a line feed (LF). You can't see it in this PR, but I can see it in the `git diff` as a `^M` character. Could that be fixed? You can read more about it [here](https://unix.stackexchange.com/questions/32001/what-is-m-and-how-do-i-get-rid-of-it). Maybe use an editor that doesn't use CRLF.

**Comment by bmbouter** [comment]
@rayanimesh Thanks again for the PR. I'd like to merge it. Are you able to push a fixed version? Thanks!

File: _posts/2015-05-21-use-docker-to-try-pulp.md (hunk 1 above)
Comment (rayanimesh) [reply _posts/2015-05-21-use-docker-to-try-pulp.md]: @bmbouter : Updated. 

I use `vim` with `ffs=unix,dos` in my `.vimrc`.  Still this file shows `^M` character. Strange.

**Comment by bmbouter** [comment]
It looks like your latest push fixed it. Thanks @rayanimesh for the fix!

**Pull request merged**